### ISSUE TITLE
Rails 8 support

### ISSUE THREAD

**Issue by xinyifly** [cgriego/active_attr#204]
```text
Resolving dependencies...
Found conflicts with dependencies with prereleases. Will retrying considering prereleases for rails, activesupport...
Retrying resolution...
Bundler::SolveFailure: Could not find compatible versions

Because every version of active_attr depends on activesupport >= 3.0.2, < 8.0
  and rails >= 8.0.0 depends on activesupport = 8.0.0,
  every version of active_attr is incompatible with rails >= 8.0.0.
So, because Gemfile depends on rails ~> 8.0
  and Gemfile depends on active_attr >= 0,
  version solving has failed.
```

**Comment by brandondrew** [comment]
@xinyifly Rails 7.2 support was only added in September of this year, so my fear is that support for Rails 8 might not be coming any time soon.

**Comment by cgriego** [comment]
Released in v0.17.1
